app/facades/quote_facade.rb

```diff
@@ -0,0 +1,6 @@
+class QuoteFacade
+  def self.quote_search(query)
+    search_results = QuoteService.quote_search(query)
+    QuoteSearch.new(search_results)
+  end
```

Comment:
This looks good. Another possible way you could build this out would be to pass two things to a single PORO (ie totalCount and Quotes). So you could do something like so:
```
response = QuotesService.quotes_by_keyword(keyword)
total_count = response[:totalCount]
 quotes = response[:results].map do |quote_data|
     Quote.new(quote_data)
 end
 QuoteResponse.new(total_count, quotes)
```